datastore/datastore.py

```diff
@@ -349,3 +354,44 @@ def exists(self):
             return elastic_search.create.exists(connection=self._client_or_connection, index_name=self._store_name)
 
         return False
+
+    def external_api_update_entity(self, dictionary_name, dictionary_data, language_script, **kwargs):
+        """
+        This method is used to populate the the entity dictionary
+        Args:
+            dictionary_name (str): Name of the dictionary that needs to be populated
+            dictionary_data (list): List of dicts consisting of value and variants
+            language_script (str): Language code for the language script used.
+            **kwargs:
+                For Elasticsearch:
+                Refer http://elasticsearch-py.readthedocs.io/en/master/helpers.html#elasticsearch.helpers.bulk
+        """
+        if self._client_or_connection is None:
+            self._connect()
+
+        if self._engine == ELASTICSEARCH:
+            self._check_doc_type_for_elasticsearch()
+            elastic_search.populate.external_api_entity_update(connection=self._client_or_connection,
+                                                               index_name=self._store_name,
+                                                               doc_type=self._connection_settings[
+                                                                    ELASTICSEARCH_DOC_TYPE],
+                                                               logger=ner_logger,
+                                                               dictionary_data=dictionary_data,
+                                                               dictionary_name=dictionary_name,
+                                                               language_script=language_script,
+                                                               **kwargs)
+
+    def transfer_entities(self, entity_list):
```

Comment:
This function is specific to elastic search.
1) Update docstrings to highlight that it work only for ES
2) Rename to specify that it is only for ES
3) If not 1 and 2 , then make it more generic if cheking the engine and connection. Confirm from Chirag to check what he thinks as well.

Reply:
Done